content/chat/rooms/typing.textile

```diff
@@ -0,0 +1,105 @@
+---
+title: Typing indicators
+meta_description: "Display typing indicators in a room so that users can see when someone else is writing a message."
+product: chat
+languages:
+  - javascript
+---
+
+Typing indicators enable you to display which users are currently writing a message in a room. This feature can be used to display a message such as __Sandi is typing...__, or when a certain threshold is reached you could instead display __Multiple people are typing...__.
+
+<aside data-type='note'>
+<p>Typing indicators need to be "enabled":/chat/rooms#create when creating or retrieving a room.</p>
+</aside>
+
+h2(#subscribe). Subscribe to typing events
+
+Subscribe to a typing events by registering a listener. Typing events can be emitted when a user starts typing, and when they stop typing. Use the "@typing.subscribe()@":https://sdk.ably.com/builds/ably-labs/ably-chat-js/main/typedoc/interfaces/Typing.html#subscribe method in a room to subscribe to receive these updates:
+
+```[javascript]
+const {unsubscribe} = room.typing.subscribe((event) => {
+  console.log(event);
+});
+```
+
+It is also possible to only subscribe to specific typing events, such as only when a user has started typing:
+
+```[javascript]
+const {unsubscribe} = room.typing.subscribe('started', (event) => {
+  console.log(${event.clientId} is currently typing...);
+});
+```
+
+h3(#payload). Typing event payload
+
+The following is an example payload of a typing event:
+
+```[json]
+{
+  "change": {
```

Comment:
The change event will no longer exist as a result of a recent PR. Only the currentlyTyping set will be returned now.